spec/jobs/yearly_review_spec.rb

```diff
@@ -189,23 +200,11 @@
         Jobs::YearlyReview.new.execute({})
         topic = Topic.last
         raw = Post.where(topic_id: topic.id).first.raw
-        expect(raw).to have_tag("table.likes-given tr.user-row-0") { with_tag("td", text: /11/) }
-        expect(raw).to have_tag("table.likes-given tr.user-row-0") {
-          with_tag("td", text: /@top_review_user/)
-        }
-        expect(raw).to have_tag("table.likes-given tr.user-row-1") { with_tag("td", text: /10/) }
-        expect(raw).to have_tag("table.likes-given tr.user-row-1") {
-          with_tag("td", text: /@reviewed_user/)
-        }
-
-        expect(raw).to have_tag("table.likes-received tr.user-row-0") { with_tag("td", text: /11/) }
-        expect(raw).to have_tag("table.likes-received tr.user-row-0") {
-          with_tag("td", text: /@reviewed_user/)
-        }
-        expect(raw).to have_tag("table.likes-received tr.user-row-1") { with_tag("td", text: /10/) }
-        expect(raw).to have_tag("table.likes-received tr.user-row-1") {
-          with_tag("td", text: /@top_review_user/)
-        }
+        expect(raw).to have_tag("div.likes-given") { with_text(/\@top_review_user\|11/) }
```

Comment:
Can those use strings, like this?

```suggestion
        expect(raw).to have_tag("div.likes-given") { with_text("@top_review_user|11") }
```

Reply:
No, we can't use it here. I'm checking whether the result _includes_ the text or not.